docs/source/kedro-viz_visualisation.md

```diff
@@ -238,4 +238,23 @@ import '@quantumblack/kedro-viz/lib/styles/styles.min.css';
 const MyApp = () => <div style={{height: `100vh`}}><KedroViz data={json} options={options} /></div>;
 ``` 
 
-For more information on how to use kedro as React component with all possible props please refer to [Kedro-Viz on npm](https://www.npmjs.com/package/@quantumblack/kedro-viz)
\ No newline at end of file
+For more information on how to use kedro as React component with all possible props please refer to [Kedro-Viz on npm](https://www.npmjs.com/package/@quantumblack/kedro-viz)
```

Comment:
```suggestion
For more information on how to use kedro as React component with all possible props, see [Kedro-Viz on NPM](https://www.npmjs.com/package/@quantumblack/kedro-viz)
```

Reply:
Is it not `npm`? I thought I see this more common, but not sure. 
![Uploading image.png…]()